rust/src/common.rs

```diff
@@ -14,6 +14,24 @@ macro_rules! take_until_and_consume (
   );
 );
 
+
+#[cfg(not(feature = "debug-validate"))]
+#[macro_export]
+macro_rules! debug_validate_bug_on (
+  ($item:expr) => {};
+);
+
+
+#[cfg(feature = "debug-validate")]
+#[macro_export]
+macro_rules! debug_validate_bug_on (
+  ($item:expr) => {
+    if $item {
+        panic!("Value should not be NULL!");
```

Comment:
`NULL` isn't right here: it triggers on `false`.

Reply:
Do you mean the panic statement should be changed to something else? It gets triggered if header is None.

Reply:
I think NULL in the string is confusing. It's about a condition being false. NULL suggests a pointer value.